lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4024,22 +4036,47 @@ bool ClangImporter::Implementation::forEachLookupTable(
 bool ClangImporter::Implementation::lookupValue(SwiftLookupTable &table,
                                                 DeclName name,
                                                 VisibleDeclConsumer &consumer) {
+
   auto &clangCtx = getClangASTContext();
   auto clangTU = clangCtx.getTranslationUnitDecl();
 
   bool declFound = false;
-
   // For operators we have to look up static member functions in addition to the
   // top-level function lookup below.
+  auto declBaseName = (SwiftContext.LangOpts.EnableCXXInterop && name.isOperator())
+          ? DeclBaseName(SwiftContext.getIdentifier("__operator" + getOperatorNameForToken(std::string{name.getBaseName().getIdentifier()})))
```

Comment:
Also nit: Seems to be missing clang-format 
You can use either git-clang-format tool that is on ${PATH_TO_SWIFT}/llvm-project/clang/tools/clang-format and you can add it to your path, and run on the git directory after `git add .`
Or clang-format -lines 1630:1660 lib/Sema/CSFix.h -i for example to run at lines in specific.

Reply:
Thanks! Will do!